bid.proto

```diff
@@ -12,8 +12,8 @@ message Bid {
     bytes nonce = 3; // BlsScalar
     StealthAddress stealth_address = 4;
     bytes commitment = 5; // JubJubCompressed
-    bytes elegibility = 6; // BlsScalar
-    bytes expiration = 7; // BlsScalar
+    fixed64 elegibility = 6; // u64
```

Comment:
The comment is redundant since you're sending a `fixed64` and not `bytes` (the comments where introduced to let a developer knows which kind of structure expects when `bytes` where sent).